content/php/concepts/arrays/terms/array_shift/array-shift.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'array_shift()'
+Description: 'Shifts/pops an element off the beginning of an array.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_shift()`** shifts/pops the first value of the array off and returns it, shortening the array by one element and shifting each element down. Numerical keys will be reset to start counting from 0, whereas the remaining literal keys will not be affected.
+
+## Syntax
+
+```pseudo
+array_shift(array &$array)
+```
+
+- `$array` is the input array.
+
+## Example
+
+The following example shifts the first element in `$counting_rhyme` and assigns the returned value to the `$removed` variable, then it prints out `$removed` and the modified `$counting_rhyme` array.
+
+```php
+<?php
+    $counting_rhyme = ["Eeny", "Meeny", "Miny", "Moe"];
+    $removed = array_shift($counting_rhyme); 
+    echo $removed . "\n"; //Prints: Eeny
+    print_r($counting_rhyme) //Prints: the modified array
```

Comment:
```suggestion
    echo $removed . "\n";
    print_r($counting_rhyme)
```

Reply:
the comments are unnecessary because the output is included below. Also update the indentations to use 2 spaces not tabs